docs/sources/.eslintrc.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     __ENV: 'readonly',
     __ITER: 'readonly',
     open: 'readonly',
+    window: 'readonly',
```

Comment:
Can you share the error you get that this is fixing?

Reply:
It addresses stuff like the following: https://github.com/grafana/k6-docs/pull/1533/files#diff-8c065fb2955bda9e9152922061225c2e47321ad1fd0229e2953087a6ae35685bR143

This file is full of them. I looked into the docs, and couldn't find any mention of `window` related to the browser module; that would explain where it comes from at least. Eslint complains this is unimported, so my "carpet bombing" solution was to essentially tell it to ignore it, I'm open to any alternative 👍🏻